docs/source/installing_dallinger_for_users.rst

```diff
@@ -91,3 +91,31 @@ you can move on.
 
 Next, you'll need :doc:`access keys for AWS, Heroku,
 etc. <aws_etc_keys>`.
+
+
+Install Heroku
+^^^^^^^^^^^^^^
+
+To run experiments locally or on the internet you need Heroku installed.
+You need at least version 3.28.0 to use Dallinger. You only need
+a Heroku account to launch experiments on the internet, you do not need it
+for local debugging.
+
+You can check your currently installed version by running:
+
+::
+
+    heroku --version
+
+Heroku is available for download from 
+`heroku.com <https://devcenter.heroku.com/articles/heroku-cli>`__.
+
+Install redis
+^^^^^^^^^^^^^
+
+Accurate debugging of experiments also requires you to have redis
```

Comment:
The requirement is stricter than this. As it stands, if Redis is installed but is not running when the experiment is being debugged, there is an error message that prevents debugging:

```
❯❯ Starting up the server...
[WARN] No ENV file found
3:39:02 PM worker.1 |  Traceback (most recent call last):
3:39:03 PM worker.1 |    File "worker.py", line 22, in <module>
3:39:03 PM worker.1 |      worker.work()
3:39:03 PM worker.1 |    File "/Users/jordansuchow/.virtualenvs/dallinger/lib/python2.7/site-packages/rq/worker.py", line 433, in work
3:39:03 PM worker.1 |      self.register_birth()
3:39:03 PM worker.1 |    File "/Users/jordansuchow/.virtualenvs/dallinger/lib/python2.7/site-packages/rq/worker.py", line 250, in register_birth
3:39:03 PM worker.1 |      if self.connection.exists(self.key) and \
3:39:03 PM worker.1 |    File "/Users/jordansuchow/.virtualenvs/dallinger/lib/python2.7/site-packages/redis/client.py", line 855, in exists
3:39:03 PM worker.1 |      return self.execute_command('EXISTS', name)
3:39:03 PM worker.1 |    File "/Users/jordansuchow/.virtualenvs/dallinger/lib/python2.7/site-packages/redis/client.py", line 578, in execute_command
3:39:03 PM worker.1 |      connection.send_command(*args)
3:39:03 PM worker.1 |    File "/Users/jordansuchow/.virtualenvs/dallinger/lib/python2.7/site-packages/redis/connection.py", line 563, in send_command
3:39:03 PM worker.1 |      self.send_packed_command(self.pack_command(*args))
3:39:03 PM worker.1 |    File "/Users/jordansuchow/.virtualenvs/dallinger/lib/python2.7/site-packages/redis/connection.py", line 538, in send_packed_command
3:39:03 PM worker.1 |      self.connect()
3:39:03 PM worker.1 |    File "/Users/jordansuchow/.virtualenvs/dallinger/lib/python2.7/site-packages/redis/connection.py", line 442, in connect
3:39:03 PM worker.1 |      raise ConnectionError(self._error_message(e))
3:39:03 PM worker.1 |  redis.exceptions.ConnectionError: Error 61 connecting to localhost:6379. Connection refused.

❯❯ Could not connect to redis instance, experiment may not behave correctly.
```

I think the ideal scenario is that `dallinger debug` starts up the Redis server if it is not already running. Another way to fix this would be to change the instructions here so that they state that Redis must be running.

(Though out-of-scope for this PR, it would be nice if there was the same behavior for the Postgres server — starting it if it isn't already started.)